.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Is this adding a pre-commit hook, would any setup be necessary in development environments? (https://github.com/typicode/husky#install ?)

Reply:
You actually don't, you just need to run `npm install`. I have added a few sections to the `README.md` on getting up and running locally and contributing etc 💯 Welcome your thoughts on this section as well 👍

Reply:
Cool, thanks!